roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/entity/Car.java

```diff
@@ -2,5 +2,63 @@
 
 import at.fhv.itm3.s2.roundabout.api.entity.ICar;
 
+import java.time.LocalTime;
+import java.util.List;
+
 public class Car implements ICar {
```

Comment:
constructor missing, for example to initialize the route list (or set the actual section)